packages/runtime/test/integration/pool-transaction-fee-test.ts

```diff
@@ -86,4 +88,44 @@ describe("Pooled Transaction Fees Test", function () {
 			RUNTIME_ERRORS.TRANSACTION.FEES_NOT_ENOUGH
 		);
 	});
+
+	it("Should unfunded accounts be able to issue transactions, through execute transaction", () => {
+		const amount = 4000;
+		const fee = 3000;
+		const johnInitialBalance = john.balance();
+		// group with fee distribution. Pooled transaction fee
+		const groupTx: types.ExecParams[] = [
+			{
+				type: types.TransactionType.TransferAlgo,
+				sign: types.SignType.SecretKey,
+				fromAccount: john.account,
+				toAccountAddr: alice.address,
+				amountMicroAlgos: amount,
+				payFlags: { totalFee: fee }, // this covers fee of entire group txns.
+			},
+			{
+				type: types.TransactionType.TransferAlgo,
+				sign: types.SignType.SecretKey,
+				fromAccount: alice.account,
+				toAccountAddr: elon.address,
+				amountMicroAlgos: amount,
+				payFlags: { totalFee: 0 }, // with 0 txn fee.
+			},
+			{
+				type: types.TransactionType.TransferAlgo,
+				sign: types.SignType.SecretKey,
+				fromAccount: elon.account,
+				toAccountAddr: john.address,
+				amountMicroAlgos: amount,
+				payFlags: { totalFee: 0 }, // with 0 txn fee.
```

Comment:
will it still work if we change the txn tan covers fee to second?

Reply:
Yes, it will be able to cover the fee of it's own txn if provided.

Reply:
lets add test for it

Reply:
Done